### DISCUSSION TITLE
Compatibility of DIYless OpenTherm Gateway

### DISCUSSION THREAD

**Discussion by ALeXXBody** [Alexwijn/SAT#42] [Q&A] [answered]
Hi all,
First of all, I want to thank you for this app, got me so excited to have it implemented in my home. 
Before discovering SAT, I ordered this gateway: https://diyless.com/product/esp8266-opentherm-gateway . thinking to use it with https://github.com/Laxilef/OTGateway as firmware and Better Thermostat as Home Assistant integration.
But SAT seems more advanced and more reliable for my set-up.

Will that gateway be compatible with SAT? Or should I find a way to order the one from Nodo-Shop?

Unfortunately, there is no UK delivery option for https://www.nodo-shop.nl/en/featured/211-opentherm-gateway.html so I am stuck with the one that I already ordered.

**Comment by critictidier** [comment]
Email Nodo Shop, they will advise you what UK delivery options they have.

**Reply by sergeantd83** [reply, marked as answer]
Hi @ALeXXBody currently we don’t support diyless board. Soon ESPHome will officially support OpenTherm for these boards. You can look the progress of this [discussion](https://github.com/esphome/esphome/discussions/7170#). We plan to support and create a new coordinator for these boards based on the official ESPHome component. Of course it will be hard because we don’t own any of them. 
Currently we support only the OTGW board and if you want to test SAT you may need to follow @critictidier recommendation.

**Reply by ALeXXBody** [reply]
Thank you for your answers. I'm waiting for Nodo Shop to reply to my email to see if it is possible to put my hands on one of these modules. 
Is quite strange that UK is not listed there, I understand that is no longer an EU country but they have other non-EU countries like India, Malaysia, Ukraine ...
